demo/fiddle/demo.details

```diff
@@ -3,6 +3,6 @@
  authors:
    - Onfido Team
  resources:
-   - https://assets.onfido.com/web-sdk-releases/6.2.0/onfido.min.js
-   - https://assets.onfido.com/web-sdk-releases/6.2.0/style.css
+   - https://assets.onfido.com/web-sdk-releases/6.3.0-rc.1/onfido.min.js
```

Comment:
why does this say rc.1? There is no specification of release candidate anywhere else in the documentation. Does this get autoupdated upon release?

Reply:
Yeap. This will get autoupdated on release when the 6.3.0 tag gets pushed to Github.